widget/embedded/src/components/SwapDetailsAlerts/SwapDetailsAlerts.Warning.tsx

```diff
@@ -37,6 +38,7 @@ export function WarningAlert(props: WaningAlertsProps) {
         title={message.shortMessage}
         action={
           <Button
+            id="widget-swap-details-warning-alert-connect-btn"
```

Comment:
I think "connect-wallet" would be a better name for the action.

Reply:
fixed